examples/tutorials/frames.py

```diff
@@ -36,14 +36,37 @@
 # and automatically determines tick labels from the plot region.
 
 fig = pygmt.Figure()
-fig.coast(shorelines="1/0.5p", region=[-180, 180, -60, 60], projection="M10i")
+fig.coast(shorelines="1/0.5p", region=[-180, 180, -60, 60], projection="M25c")
 fig.basemap(frame="a")
 fig.show()
 
 ########################################################################################
 # Add automatic grid lines to the plot by adding a ``g`` to ``frame``:
 
 fig = pygmt.Figure()
-fig.coast(shorelines="1/0.5p", region=[-180, 180, -60, 60], projection="M10i")
+fig.coast(shorelines="1/0.5p", region=[-180, 180, -60, 60], projection="M25c")
 fig.basemap(frame="ag")
 fig.show()
+
+########################################################################################
+# Title
+# -----
+#
+# The figure title can be set by passing ``"+t<title>"`` to the ``frame`` parameter of
+# :meth:`pygmt.Figure.basemap`. Passing multiple arguments to ``frame`` can be done by
+# using a list, as show in the example below.
+
+fig = pygmt.Figure()
+fig.coast(shorelines="1/0.5p", region="IS", projection="M25c")
+fig.basemap(frame=["a", "+tIceland"])
+fig.show()
+
+########################################################################################
+# To use a title with multiple words, the title must be placed inside another set of
+# quotation marks. To prevent the quotation marks from appearing in the figure title,
+# the ``"\""`` escape character notation can be used to cancel out the quotation marks.
+
+fig = pygmt.Figure()
+fig.coast(shorelines="1/0.5p", region="TT", projection="M25c")
+fig.basemap(frame=["a", "+t\"Trinidad and Tobago\""])
```

Comment:
An alternative way (which avoids backslashes) is to use single-quotes `'` outside, with double-quotes `"` inside. See also #247 (which we really need to figure out a proper solution for).

```suggestion
fig.basemap(frame=["a", '+t"Trinidad and Tobago"'])
```